src/pages/install/master.md

```diff
@@ -0,0 +1,43 @@
+
+# Using RGBDS `master`
+
+`master` is the name of RGBDS' main development branch.
+Using a "master" version of RGBDS means using an in-development version: this means you get access to the newest features faster (and you help test them!), but it also means that they have undergone a little less testing.
+
+We are very grateful to all who takes the time to use `master` versions; all of you help make the releases better for everyone.
+But, if you don't want to, you can [go back to the install page](/install).
+
+
+## Using a package manager
+
+Your package manager *may* have a `master` version of RGBDS available, such as Arch's [`rgbds-git` AUR package](https://aur.archlinux.org/packages/rgbds-git).
+If that is the case, it is recommended to use that, as it should be simpler and integrate better with the rest of your system.
+
+As this is highly OS-specific, we cannot provide any generic instructions.
+Sorry!
+
+## Building it yourself
+
+:::tip
+
+If your package manager is not an option, this is the preferred way.
+Unless you are on Windows, because development on Windows is hell; consider [using our CI](#using-our-ci) instead.
```

Comment:
```suggestion
Unless you are on Windows, consider [using our CI](#using-our-ci) instead.
```
I would avoid harsh/sarcastic comments here, they really don't help anyone

Reply:
I've had another series of bad times with Windows yesterday :upside_down_face:

Reply:
avivace's suggested change inverts the meaning. It was "If Windows then use our CI." The suggested change changes it to "If *not* Windows then use our CI." Try "Unless you are on Windows, in which case consider...".

Reply:
Right, didn't really re read - my point there is avoid those kind of remarks

Reply:
Part of the intention was to justify why Windows requires heading a different way. What's suggested front-and-center is "compile it yourself", which is a massive PITA on Windows, and I think warning of that upfront is preferable to the user finding out about all that friction on their own, and giving up.

Reply:
github CI can do windows builds can't it?

Reply:
Yes, that's exactly what the paragraph is directing readers towards.

Reply:
Sorry, the start of the sentence made me think that Windows *is not* supported by the CI builds

Reply:
Yeah, as Pino mentioned, it's now backwards, lol